sktime/forecasting/base/adapters/_neuralforecast.py

```diff
@@ -137,66 +139,73 @@ def algorithm_parameters(self: "_NeuralForecastAdapter") -> dict:
         - custom model name (``alias``) - used from ``algorithm_name``
         """
 
-    def _get_valid_parameters(self: "_NeuralForecastAdapter") -> dict:
-        """Get valid parameters for the underlying NeuralForecast algorithm class.
+    def _ignore_invalid_parameters(self: "_NeuralForecastAdapter") -> dict:
```

Comment:
Why don't you use Dict from typing?

Reply:
`Dict` is deprecated and `dict` is recommended.

https://docs.python.org/3/library/typing.html#typing.Dict